### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.resource.managedapplications.models.error_response.ErrorResponse
  - azure.mgmt.resource.managedapplications.models.error_response.ErrorResponseException
  fullName: azure.mgmt.resource.managedapplications.models.error_response
  langs:
  - python
  module: azure.mgmt.resource.managedapplications.models.error_response
  name: error_response
  source:
    id: error_response
    path: azure-mgmt-resource\azure\mgmt\resource\managedapplications\models\error_response.py
    remote:
      branch: master
      path: azure-mgmt-resource\azure\mgmt\resource\managedapplications\models\error_response.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.mgmt.resource.managedapplications.models.error_response
references:
- fullName: azure.mgmt.resource.managedapplications.models.error_response.ErrorResponse
  isExternal: false
  name: ErrorResponse
  parent: azure.mgmt.resource.managedapplications.models.error_response
  uid: azure.mgmt.resource.managedapplications.models.error_response.ErrorResponse
- fullName: azure.mgmt.resource.managedapplications.models.error_response.ErrorResponseException
  isExternal: false
  name: ErrorResponseException
  parent: azure.mgmt.resource.managedapplications.models.error_response
  uid: azure.mgmt.resource.managedapplications.models.error_response.ErrorResponseException
